test/sequential/test-net-connect-local-error.js

```diff
@@ -3,28 +3,21 @@ const common = require('../common');
 const assert = require('assert');
 const net = require('net');
 
+const expectedErrorCodes = ['ECONNREFUSED', 'EADDRINUSE'];
```

Comment:
I need your advice on a proper comment for this. Is this ok? 
```
// EADDRINUSE is expected to occur on FreeBSD
// Please see https://github.com/nodejs/node/issues/13055 for more details
```

Not really sure if you guys approve of URLs in code comments.